lighthouse/classes/events/beckman/source_completed.py

```diff
@@ -0,0 +1,83 @@
+import logging
+from typing import Dict
+
+from flask import current_app as app
+
+from lighthouse.classes.events.source_plate_event import SourcePlateEvent
+from lighthouse.classes.mixins.labwhere import LabWhereMixin
+from lighthouse.helpers import mongo
+from lighthouse.messages.broker import Broker
+from lighthouse.messages.message import Message
+
+logger = logging.getLogger(__name__)
+
+
+class Completed(SourcePlateEvent, LabWhereMixin):
+    @property
+    def robot_serial_number(self) -> str:
+        return self._robot_serial_number
+
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
```

Comment:
This line will go to the __init__ of the classes we inherit, which can get complex through multiple inheritance; we are inheriting from LabwareMixin, which I think shouldnt be in the chain of inheritance as it is a mixin module. 
I would use LabwareMixin as a class instead, instantiate it in an attribute and use it when needed:
```
self._labwhere = LabwareMixin()
```

Reply:
Some more thought and research needed for this.